### PR TITLE
[GPUImage] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: iOS/GPUImage/component/component.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: 0.1.7.1
+version: 0.1.7
```
Comment (bholmes) [inline iOS/GPUImage/component/component.yaml]: 0.1.7.2

File: iOS/GPUImage/nuget/GPUImage.nuspec [hunk 2]
```diff
@@ -3,14 +3,14 @@
   <metadata>
     <id>GPUImage</id>
     <title>GPUImage</title>
-    <version>0.1.7.1</version>
+    <version>0.1.7</version>
```
Comment (bholmes) [inline iOS/GPUImage/nuget/GPUImage.nuspec]: 0.1.7.2

File: iOS/GPUImage/nuget/GPUImage.nuspec [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>An open source iOS framework for GPU-based image and video processing.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
-    <projectUrl>https://go.microsoft.com/fwlink/?linkid=865063</projectUrl>
-    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=864973</licenseUrl>
+    <projectUrl>https://components.xamarin.com/view/GPUImage</projectUrl>
```
Comment (bholmes) [inline iOS/GPUImage/nuget/GPUImage.nuspec]: Go back to using the FWLinks.  However they will need to be updated in the FWLink tool to point to GitHub.

File: iOS/GPUImage/build.cake [hunk 4]
```diff
@@ -4,7 +4,7 @@
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
 var IOS_PODS = new List<string> {
-	"platform :ios, '5.0'",
+	"platform :ios, '11.2'",
```
Comment (SotoiGhost) [inline iOS/GPUImage/build.cake]: Does this pod support only the most recent iOS version?

File: iOS/GPUImage/nuget/GPUImage.nuspec [hunk 5]
```diff
@@ -3,14 +3,14 @@
...
+    <licenseUrl>https://components.xamarin.com/license/gpuimage</licenseUrl>
```
Comment (newky2k) [inline iOS/GPUImage/nuget/GPUImage.nuspec]: needs to use FWLinks -> https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15

File: iOS/GPUImage/nuget/GPUImage.nuspec [hunk 6]
```diff
@@ -3,14 +3,14 @@
   <metadata>
     <id>GPUImage</id>
     <title>GPUImage</title>
-    <version>0.1.7.1</version>
```
Comment (newky2k) [inline iOS/GPUImage/nuget/GPUImage.nuspec]: Also need to change the package reference  to 0.1.7.2

**Review by newky2k — CHANGES_REQUESTED** [review]

File: iOS/GPUImage/nuget/GPUImage.nuspec (hunk 2 above)
Comment (newky2k) [reply iOS/GPUImage/nuget/GPUImage.nuspec]: Also need to change the packages version that is referenced to 0.1.7.2

**Review by newky2k — CHANGES_REQUESTED** [review]
Updates need

File: iOS/GPUImage/nuget/GPUImage.nuspec [hunk 7]
```diff
@@ -3,14 +3,14 @@
...
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>An open source iOS framework for GPU-based image and video processing.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
-    <projectUrl>https://go.microsoft.com/fwlink/?linkid=865063</projectUrl>
-    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=864973</licenseUrl>
+    <projectUrl>https://components.xamarin.com/view/GPUImage</projectUrl>
+    <licenseUrl>https://components.xamarin.com/license/gpuimage</licenseUrl>
   </metadata>
   <files>
     <file src="output/unified/GPUImage.dll" target="lib/Xamarin.iOS" />
```
Comment (newky2k) [inline iOS/GPUImage/nuget/GPUImage.nuspec]: You need to add a the external dependecies file as an extra and update it with the correct copyright notice
like this -> https://github.com/newky2k/XamarinComponents/blob/a00b2b16339b96acf55359228b6a7ab0f5058e10/iOS/HDNotification/nuget/Xamarin.HDNotification.nuspec

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — APPROVED** [review]

File: iOS/GPUImage/component/component.yaml [hunk 8]
```diff
@@ -13,7 +13,11 @@ libraries:
...
+    installNuGets:
+      - project: GPUImageSample
+        packages:
+          - Xamarin.GPUImage
 no_build: true
 is_shell: true
 packages:
-  ios-unified: GPUImage, Version=0.1.7.1
\ No newline at end of file
+  ios-unified: GPUImage, Version=0.1.7.2
```
Comment (bholmes) [inline iOS/GPUImage/component/component.yaml]: Missing the Third Party Notices file

```
additional-files:
  - source: "../External-Dependency-Info.txt"
    destination: "THIRD-PARTY-NOTICES.txt"
```

File: iOS/GPUImage/component/component.yaml [hunk 9]
```diff
@@ -13,7 +13,11 @@ libraries:
 samples:
   - name: "GPUImage Sample"
     path:  ../samples/GPUImageSample.sln
+    installNuGets:
+      - project: GPUImageSample
+        packages:
+          - Xamarin.GPUImage
```
Comment (bholmes) [inline iOS/GPUImage/component/component.yaml]: `GPUImage` not `Xamarin.GPUImage`

File: iOS/GPUImage/component/component.yaml [hunk 10]
```diff
@@ -13,7 +13,11 @@ libraries:
...
 no_build: true
 is_shell: true
```
Comment (bholmes) [inline iOS/GPUImage/component/component.yaml:24]: missing `local-nuget-repo: ../output`

**Review by bholmes — CHANGES_REQUESTED** [review]

File: iOS/GPUImage/component/component.yaml [hunk 11]
```diff
@@ -13,7 +13,11 @@ libraries:
 samples:
   - name: "GPUImage Sample"
     path:  ../samples/GPUImageSample.sln
+    installNuGets:
```
Comment (newky2k) [inline iOS/GPUImage/component/component.yaml]: missing the remove project section to remove the source project reference

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — CHANGES_REQUESTED** [review]
The licence.md file does not exists in the root folder so the component will not compile

File: iOS/GPUImage/component/component.yaml [hunk 12]
```diff
@@ -13,7 +14,18 @@ libraries:
...
+      - project: GPUImageSample
+        packages:
+          - Xamarin.GPUImage
 no_build: true
 is_shell: true
 packages:
-  ios-unified: GPUImage, Version=0.1.7.1
\ No newline at end of file
+  ios-unified: GPUImage, Version=0.1.7.
```
Comment (newky2k) [inline iOS/GPUImage/component/component.yaml:26]: there is an issues with this version number.  Its missing the last digit of the version number

**Review by newky2k — CHANGES_REQUESTED** [review]

**Comment by newky2k** [comment]
Closed refer to #383

**Pull request closed without merging**